.github/workflows/build.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           ref: master
 
       - name: Setup env
-        run: sh ./setup_env.sh
+        run: sh ./setup_env.sh github
```

Comment:
@johnschoeman the `build` work flow passes `github` to the setup script, which is a no-op